src/app/models/form.server.model.ts

```diff
@@ -463,15 +462,15 @@ const compileFormModel = (db: Mongoose): IFormModel => {
             // mode case, or else we may accidentally set Singpass/Corppass/SGID
             // storage mode forms to FormAuthType.NIL
           } else if (
-            [FormAuthType.SP, FormAuthType.CP, FormAuthType.SGID].includes(v)
+            [
```

Comment:
We might not need to check for FormAuthType anymore since email + storage are equally supporting all auth types.

Reply:
yup you're right! changed here [a74abcf](https://github.com/opengovsg/FormSG/pull/6870/commits/a74abcf3a9d57e93d17e4d5b9963cacc206f4f34)